pom.xml

```diff
@@ -445,8 +451,10 @@
                   <DB_PASSWORD>${test.db.password}</DB_PASSWORD>
                   <DB_PORT>${test.db.port}</DB_PORT>
                   <HAUTH_VERSION>${project.version}</HAUTH_VERSION>
-                  <SECRET_KEY_GENERATION_PASSWORD>${test.secretkey.password}</SECRET_KEY_GENERATION_PASSWORD>
-                  <SECRET_KEY_GENERATION_SALT>${test.secretkey.salt}</SECRET_KEY_GENERATION_SALT>
+                  <UCLA_COOKIE_SECRET_KEY_GENERATION_PASSWORD>${test.secretkey.password}</UCLA_COOKIE_SECRET_KEY_GENERATION_PASSWORD>
```

Comment:
I think this was renamed to UCLA* to sort of counterbalance having the Sinai* ENVs, but isn't this a generic password/salt that any user/institution could use?

It might make more sense to just add the Sinai specific stuff without renaming the old?

Reply:
I agree that the renaming isn't really necessary. Will revert UCLA branding.